portfolio/src/main/webapp/script.js

```diff
@@ -156,4 +156,38 @@ function deleteAll() {
         );
   })
   location.reload();
-}
\ No newline at end of file
+}
+
+function fetchBlobstoreUrlAndShowImage() {
+  fetch('/blobstore-upload-url')
+  .then((response) => {
+    return response.text();
+  })
+  .then((imageUploadUrl) => {
+    const messageForm = document.getElementById('my-form');
+    messageForm.action = imageUploadUrl;
+    messageForm.classList.remove('hidden');
+    getImageSRC();
+  });
+}
+
+function getImageSRC() {
+  fetch('/my-form-handler')
+  .then(response => response.json())
+  .then((images) => {
+    if(images.length == checkIfAnElementIsEmpty ){
+      //This is not an error, just to show the user that it is waiting for an image to be uploaded
+      throw new Error('Image is not uploaded yet');
```

Comment:
What is the goal here? Do you intend to show this message to the user?

Reply:
The goal is from them to see it in the console and also, for the code to append an image not to run until there is an image tag inside the appropriate place

Reply:
Hmm. What use case are you imagining users checking the console for failures? What it be more useful to display the failure message to the screen somehow?

Reply:
the way the page gets the picture is whenever the page is loaded up it sees if there is an image inside the array and puts it on the screen. By putting an alert it would give the warning before the user even has a chance to upload an image